doc/guide.rst

```diff
@@ -750,3 +750,53 @@ an input to the above mentioned genotyping commands. This can be done by running
     whatshap find_snv_candidates ref.fasta input.bam -o variants.vcf
 
 If Nanopore reads are used for calling SNPs, it is recommended to add option --nanopore to the above command.
+
+Polyploid Phasing
+=================
+
+In addition to diploid phasing, WhatsHap also supports polyploid phasing 
+through a different algorithm. The ``whatshap polyphase`` command works 
+almost the same as the ``phase`` command with a few restrictions:
+
+1. An additional integer argument ``--ploidy`` must be specified. This ploidy
+must match the ploidy in the provided VCF file(s). The ploidy also greatly
+impacts the running time as the phasing becomes more complex. Ploidies
+higher than 6 may take very long to process.
+
+2. WhatsHap will use available genotype information from the VCF file(s), but
+the computed haplotypes are not guaranteed to follow these genotypes, if they
+deviate too much from the allele distribution among the aligned reads.
+
+3. Polyploid phasing on pedigrees is not supported yet.
+
+4. Structural variants are not considered during the phasing. The algorithm
+will always produce the provided number of haplotypes at any location.
+
+Opposed to the diploid phasing algorithm, there is no strict limitation
+regarding the coverage of the input reads. However, it still increases the
+running time and we do not recommend to use more than 120X. In principle it is
+possible to phase diploid samples via the ``polyphase`` command, but the
+results will likely be less accurate than the diploid phasing mode, as the
+latter is more specialized for the diploid case.
+
+A problem, which occurs in polyploid phasings is the connectivity of phased
+variants. The more haplotypes exist the more reads are required between
+consecutive variants to properly connect haplotype fragments from both sides.
```

Comment:
```suggestion
To achieve reliable phasing, as many haplotypes as possible should be represented
in the input reads. In case of unrepresented haplotypes, phasing can become impossible
and the output haplotypes are broken into phased blocks.
```